ModulemdTranslationHelpers/Utils.py

```diff
@@ -11,6 +11,100 @@
     sys.exit(77)
 
 
+def get_latest_modules_in_tag(session, tag, debug=False):
+    """
+    Get the most-recently built versions of each (module,stream) pair from
+    a Koji tag
+    :param session: A Koji session
+    :param tag: A koji tag
+    :return: A list of the most recent build of all modules in the tag.
+    """
+
+    # Koji sometimes disconnects for no apparent reason. Retry up to 5
+    # times before failing.
+    for attempt in range(5):
+        try:
+            tagged = session.listTagged(tag)
+        except requests.exceptions.ConnectionError:
+            if debug:
+                print("Connection lost while retrieving builds for tag %s, "
+                      "retrying..." % tag,
+                      file=sys.stderr)
+        else:
+            # Succeeded this time, so break out of the loop
+            break
+
+    # Find the latest, in module terms.  Pungi does this.
+    # Collect all contexts that share the same NSV.
+    NSVs = {}
+    for entry in tagged:
+        name, stream = entry['name'], entry['version']
+        version = entry['release'].rsplit('.', 1)[0]
+
+        NSVs[name] = NSVs.get(name, {})
+        NSVs[name][stream] = NSVs[name].get(stream, {})
+        NSVs[name][stream][version] = NSVs[name][stream].get(version, [])
+        NSVs[name][stream][version].append(entry)
+
+    latest = []
+    for name in NSVs:
+        for stream in NSVs[name]:
+            version = sorted(list(NSVs[name][stream].keys()))[-1]
+            latest.extend(NSVs[name][stream][version])
+
+    return latest
+
+
+def get_index_from_tags(session, tags, debug=False):
```

Comment:
```suggestion
def get_index_from_tags(session, tags):
```

Use the python logging facility instead.